### PR TITLE
OSDOCS#6141:Added policy info for CSI drivers role in ROSA

### PR DESCRIPTION
**Pull request by AedinC**
Version(s):
4.14+

Issue:
[OSDOCS-6141](https://issues.redhat.com/browse/OSDOCS-6141)

Link to docs preview:
[Creating a cluster with customizations by using OpenShift Cluster Manager](https://68399--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations#rosa-sts-creating-cluster-customizations-ocm_rosa-sts-creating-a-cluster-with-customizations)

[Creating a cluster with customizations using the CLI](https://68399--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations#rosa-sts-creating-cluster-customizations-cli_rosa-sts-creating-a-cluster-with-customizations)

QE review:
- [x] QE has approved this change.

Peer review:
- [x] Peer reviewer has approved this change.

### CODE REVIEW

File: modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc [hunk 1]
```diff
@@ -29,10 +29,14 @@ Only public and AWS PrivateLink clusters are supported with STS. Regular private
 ====
 The EBS operator role is required in addition to the account roles to successfully create your cluster.
```
Comment (skrthomas) [inline modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc:30]: noticed that this may need to have an O->Operator

**Review by skrthomas — COMMENTED** [review]
/lgtm

File: modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc [hunk 2]
```diff
@@ -27,12 +27,16 @@ Only public and AWS PrivateLink clusters are supported with STS. Regular private
 +
 [IMPORTANT]
 ====
-The EBS operator role is required in addition to the account roles to successfully create your cluster.
+The EBS Operator role is required in addition to the account roles to successfully create your cluster.
+
+This role must be attached with the `ManagedOpenShift-openshift-cluster-csi-drivers-ebs-cloud-credentials` policy, an IAM policy required by ROSA to manage back-end storage through the Container Storage Interface (CSI).
+
+For more information about the policies and permissions that the cluster Operators require, see xref:../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-account-wide-roles-and-policies-creation-methods_rosa-sts-about-iam-resources[Methods of account-wide role creation].
```
Comment (michaelryanpeter) [inline modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc:34]: We can't use `xrefs` in modules. Please refer to the OpenShift docs [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references).

If you want to call out a link for customers to follow in a module, you refer to the title of the link in quotes, then add [a trailing include additional resources section](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-assemblies) with the `xref` in the assembly.

If you run into to trouble, members of your team can help you out. Or you can ask for help in the team Slack channel.

File: modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc [hunk 3]
```diff
@@ -335,11 +339,15 @@ If you specified custom ARN paths when you created the associated account-wide r
...
```
Comment (michaelryanpeter) [inline modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc:346]: See the comment above about `xrefs` in modules.

File: modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc [hunk 4]
```diff
@@ -219,12 +219,16 @@ PVs created by using any other storage class are only encrypted if the storage c
 +
 [IMPORTANT]
 ====
-The EBS operator role is required in addition to the account roles to successfully create your cluster.
+The EBS Operator role is required in addition to the account roles to successfully create your cluster.
+
+This role must be attached with the `ManagedOpenShift-openshift-cluster-csi-drivers-ebs-cloud-credentials` policy, an IAM policy required by ROSA to manage back-end storage through the Container Storage Interface (CSI).
+
+For more information about the policies and permissions that the cluster Operators require, see xref:../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-account-wide-roles-and-policies-creation-methods_rosa-sts-about-iam-resources[Methods of account-wide role creation].
```
Comment (michaelryanpeter) [inline modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc:226]: See the comment above about `xrefs` in modules.

File: modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc [hunk 5]
```diff
@@ -417,11 +421,16 @@ If you opted to use *Auto* mode, {cluster-manager} creates the Operator roles an
...
```
Comment (michaelryanpeter) [inline modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc:428]: See the comment above about `xrefs` in modules.

**Review by michaelryanpeter — COMMENTED** [review]
The content LGTM, but the `xrefs` must be removed from the modules before we can merge. 

On a side note, you might want to consider using a [text snippet](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-text-snippets) for the repeated admonition.

/remove-label merge-review-needed
/remove-label merge-review-in-progress

File: modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc (hunk 2 above)
Comment (AedinC) [reply modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc:34]: Thanks @michaelryanpeter, I will fix these.

Comment (skrthomas) [reply modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc:34]: Oh man, sorry for missing this in my peer review @michaelryanpeter and @AedinC .

Comment (AedinC) [reply modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc]: I can't believe I forgot about it too @skrthomas!

Comment (michaelryanpeter) [reply modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc]: Not a problem, at all! I forget sometimes. :)

My kingdom for an xref...

**Pull request merged**